requirements.txt

```diff
@@ -4,12 +4,14 @@ Pypubsub==4.0.3
 configparser==5.0.1
 h5py==2.10.0
 imageio==2.9.0
+mido==1.2.10
```

Comment:
I think it's better to mark this dependency as optional. You change to something like:

```
mido[navigator]==1.2.10
```

Or create a new requirements file, something like `navigator_requirements.txt"

Reply:
Thank you. I moved them to a separate file (namely, `optional-requirements.txt`, as they're not strictly required for navigator, either).

Please note that this change conflicts slightly with the changes in my other PR (#282), which also add optional-requirements.txt file, but I will resolve the conflict once one of the PRs is merged.